web-client/src/ustc-ui/Tabs/Tabs.jsx

```diff
@@ -88,6 +89,11 @@ const renderTabFactory = ({
               <div className="icon-tab-notification-exclamation">!</div>
             </div>
           )}
+          {draftCount !== '0' && (
+            <div className="icon-tab-unread-messages">
```

Comment:
This comment is optional, but I wanted to point it out: 

It feels strange that this generic Tabs component has logic related to notifications and unread messages.  I know we're following the example on line 87, but I think this might warrant a devex refactor to have any additional icons passed in as a prop instead of having logic specific to notifications / draftCounts.

Reply:
We will create a DEVEX task to address this feedback!